test_cli_args.py

```diff
@@ -59,18 +59,20 @@ def test_center_exclude_arg_should_filter_excluded_centers(MockDoctolibDE, tmp_p
     assert mock_doctolib_de.get_patients.called
     assert mock_doctolib_de.try_to_book.called
     for call_args_list in mock_doctolib_de.try_to_book.call_args_list:
-        assert call_args_list.args[0]['name_with_title'] != excluded_center
-        assert call_args_list.args[0]['city'] == city
+        assert call_args_list[0][0]['name_with_title'] != excluded_center
+        assert call_args_list[0][0]['city'] == city
```

Comment:
Why?

Reply:
In Python 3 - getting `Fail: TypeError: string indices must be integers`